src/CaBundle.php

```diff
@@ -160,6 +160,7 @@ public static function validateCaFile($filename, LoggerInterface $logger = null)
         }
 
         $contents = file_get_contents($filename);
+        $contents = preg_replace("/^(\\-+(?:BEGIN|END))\\s+TRUSTED\\s+(CERTIFICATE\\-+)\$/m", '$1 $2', $contents);
```

Comment:
Actually this could be moved right before the openssl_x509_parse call, as it's not really needed otherwise?